source/dmocks/method_mock.d

```diff
@@ -66,15 +68,19 @@ string BuildForwardCall(string methodName, bool dynamicVarArgs)()
 
     return `static if (is(typeof(mocked___.` ~ methodString ~ argsPassed ~ `)))
             {
-                return (mocked___.` ~ methodString ~ argsPassed ~ `);
+                return (mocked___.`
+        ~ methodString ~ argsPassed ~ `);
```

Comment:
bug of "keep line breaks"?

Reply:
Kann sein, es ist plausibel dass er da durcheinander kommt, vielleicht denkt er wegen dem line break im String dass die Zeile über 120 Zeichen hinausgeht.

Reply:
https://github.com/dlang-community/dfmt/issues/476